lib/embulk/output/bigquery.rb

```diff
@@ -227,14 +228,55 @@ def self.configure(config, schema, task_count)
           task['abort_on_error'] = (task['max_bad_records'] == 0)
         end
 
+        if task['time_partitioning'] && task['range_partitioning']
+          raise ConfigError.new "`time_partitioning` and `range_partitioning` cannot be used at the same time"
+        end
+
         if task['time_partitioning']
           unless task['time_partitioning']['type']
             raise ConfigError.new "`time_partitioning` must have `type` key"
           end
-        elsif Helper.has_partition_decorator?(task['table'])
+        end
+
+        if Helper.has_partition_decorator?(task['table'])
+          if task['range_partitioning']
+            raise ConfigError.new "Partition decorators(`table_name$20250101`) don't support `range_partition`"
```

Comment:
Which do you think is better?

* fix table name: `table_name$20250101`
* From user config `#{task['table']}`
* Removing example: `Partition decorators don't support `range_partition`

```suggestion
            raise ConfigError.new "Partition decorators(`#{task['table']}`) don't support `range_partition`"
```